docs/content/get-started/set-up.md

```diff
@@ -63,19 +63,21 @@ The SpatialOS GDK for Unity source code is hosted on GitHub. You need to downloa
 
  **Using Visual Studio?**
 
-As you install [Visual Studio](https://www.visualstudio.com/downloads/), click on the **Workloads** tab in the Installer. If you already have Visual Studio installed, you can find this by clicking on the **More** option for *Visual Studio Build Tools* in the Visual Studio Installer, and selecting **Modify** from the drop-down menu.
+You need to install the **.NET Core cross-platform development** and **Game development with Unity** workloads. To to this:
+
+- As you install [Visual Studio](https://www.visualstudio.com/downloads/), click on the **Workloads** tab in the Installer. If you already have Visual Studio installed, you can find the **Workloads** tab by opening Visual Studio Installer and, in the Products section, clicking the **Modify** option for Visual Studio 2017. If the **Modify** option is not visible, this option may also be found in the drop-down menu that appears when you select the **More** option.
```

Comment:
If "Products" is part of the UI, it's bolded.

```suggestion
- As you install [Visual Studio](https://www.visualstudio.com/downloads/), click on the **Workloads** tab in the Installer. If you already have Visual Studio installed, you can find the **Workloads** tab by opening Visual Studio Installer and, in the **Products** section, selecting **Modify** for Visual Studio 2017. If you can't see the **Modify** option is not visible, select the **More**.
```
"Select" not "click on" or "click".
Avoid gerunds where possible and use imperatives (instructions).